Server/mods/deathmatch/logic/CPlayer.h

```diff
@@ -351,6 +352,7 @@ class CPlayer : public CPed, public CClient
     uint                                        m_uiD3d9Size;
     SString                                     m_strD3d9Md5;
     SString                                     m_strD3d9Sha256;
+    bool                                        bIsQuit;

```

Comment:
Member variables must be prefixed with **m_**. Furthermore, you might want to name this variable **m_bIsLeaving**, but this is up for debate.